icubam/predicu/plot/__init__.py

```diff
@@ -103,74 +113,60 @@ def plot_int(
     PLOTS.append(plot_name)
 
 
-def plot(plot_name, data, **plot_args):
+def plot(
+  plot_name: str,
+  cached_data: Dict[str, pd.DataFrame],
+  output_dir: str = DEFAULTS["output_dir"],
+  output_type: str = DEFAULTS["output_type"],
+  api_key: str = DEFAULTS["api_key"],
+  matplotlib_style: str = DEFAULTS["matplotlib_style"],
```

Comment:
The default are a bit convoluted. If you change the `DEFAULTS` dict globally and then use this functions, the parameter defaults will likely not change.

Maybe instead set default to None, and then,
```py
if api_key is None:
    api_key = DEFAULTS["api_key"]
```
or even better just remove the `DEFAULTS` dict and write them plainly in these 2 functions. Most are None anyway.

Reply:
done